src/Cli/UpCommand.php

```diff
@@ -39,16 +40,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             return 1;
         }
-        try {
-            $dockerComposePath = $this->getDockerComposePath($processRunner);
-            pcntl_exec($dockerComposePath, ['up']);
 
-            return 0;
+        $userInteraction->writeTitle('Starting application containers');
+
+        try {
+            $processRunner->run(new Process('docker-compose up -d'));
+            $userInteraction->writeTitle('Application containers successfully started');
         } catch (ProcessFailedException $e) {
+            echo $e->getProcess()->getOutput();
+
             return 1;
         }
+
+        // Display running containers
```

Comment:
Yes I prefer this it makes it explicit that it is the same as the output for that command. 

I think you can do something like `$command = $this->getApplication()->find('ps');` which doesn't make much difference at the moment but will when there are constructor arguments for the PS command.